homeassistant/components/calendar/icons.json

```diff
@@ -0,0 +1,12 @@
+{
+  "entity_component": {
+    "_": {
+      "default": "mdi:calendar"
```

Comment:
```suggestion
      "default": "mdi:calendar",
      "state": {
        "on": "mdi:calendar-check",
        "off": "mdi:calendar-blank"
      }
```

Reply:
That is a nice improvement, but it doesn't seem to match what we have right now?

Reply:
True, but since this is possible now, why not improve and use it? Similar to https://github.com/home-assistant/core/pull/108417.

Reply:
This is an migration, not an adjustment.

Reply:
I don't get it, sorry. Should I create a separate PR for this?

Reply:
Yes please 👍

Reply:
@jrieger Sorry, yeah, sure! These PRs are primarily aimed at migrating / putting in place the icon translations with the least amount of user-facing changes.